src/debugger/direct/directDebugSession.ts

```diff
@@ -46,6 +46,31 @@ export class DirectDebugSession extends DebugSessionBase {
             .then(() => {
                 logger.log("Launching the application");
                 logger.verbose(`Launching the application: ${JSON.stringify(launchArgs, null , 2)}`);
+<<<<<<< HEAD
+                return ProjectVersionHelper.getReactNativeVersions(launchArgs.cwd, launchArgs.platform === "windows")
+                    .then(versions => {
+                        extProps = TelemetryHelper.addPropertyToTelemetryProperties(versions.reactNativeVersion, "reactNativeVersion", extProps);
+                        if (launchArgs.platform === "windows") {
+                            extProps = TelemetryHelper.addPropertyToTelemetryProperties(versions.reactNativeWindowsVersion, "reactNativeWindowsVersion", extProps);
+                        }
+                        return new Promise(() => {
```

Comment:
Is this a temporary change until the telemetry part is migrated to promises approach?

Reply:
Yes, it will be replaced later.